twake/frontend/src/app/components/OnBoarding/AccountStatusComponent.tsx

```diff
@@ -1,32 +1,25 @@
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 
 import Banner from 'app/components/Banner/Banner';
 import { Button, Typography } from 'antd';
 import Languages from 'services/languages/languages.js';
 import ModalManager from 'app/components/Modal/ModalManager';
 import UnverifiedAccount from './popups/UnverifiedAccount';
 import BlockedAccount from './popups/BlockedAccount';
+import UserService from 'services/user/user.js';
+import InitService from 'app/services/InitService';
 
 const AccountStatusComponent = (): JSX.Element => {
-  const [period, setPeriod] = useState<number>(8);
-  const [displayBanner, setDisplayBanner] = useState<boolean>(
-    period < 7 && period > 0 ? true : false,
-  );
-
-  useEffect(() => {
-    isBlockedAccount();
-  }, [period]);
+  const user = UserService.getCurrentUser();
+  const maxUnverifiedDays: number = 7;
```

Comment:
You do not need `:number` when you initialize with number, this is implicit type

Reply:
removed